### PR TITLE
fix(justfile): Fixes "wrong password" issue with generated certs

### PR DESCRIPTION
**Pull request by thomastaylor312**
Based on testing from the team and what I found on the internet (always
a reliable source of information), having an empty password with a
pkcs bundle can cause errors. This exports a default password to be
used with the certificate.

Also, I removed the kubelet=debug setting for logging because kubelet is
a _really_ verbose default

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-export RUST_LOG := "wascc_host=debug,wascc_provider=debug,wasi_provider=debug,main=debug,kubelet=debug"
+export RUST_LOG := "wascc_host=debug,wascc_provider=debug,wasi_provider=debug,main=debug"
```
Comment (rylev) [inline justfile]: Why did you remove the `kubelet` log level? too noisy?

Comment (thomastaylor312) [reply justfile]: > Also, I removed the kubelet=debug setting for logging because kubelet is
a really verbose default
